dask/show_versions.py

```diff
@@ -0,0 +1,57 @@
+from json import dumps
+from platform import uname
+from sys import stdout, version_info
+
+from pandas._typing import JSONSerializable
+from pandas.compat._optional import get_version, import_optional_dependency
+
+
+def show_versions(as_json: bool = False) -> None:
+    """Provide version information for bug reports.
+
+    Parameters
+    ----------
+    as_json : bool, default False
+        * If False, outputs info in a YAML format to the console (can be copy-
+          pasted into an environment.yml file for conda).
+        * If True, outputs info in JSON format to the console.
```

Comment:
Maybe let's hold off on adding this flag for now until someone asks for it (if they do) with a reasonably generic use case. I think just a plain `dask.show_versions()` call that prints to the screen is a good first step.